library/libraries.js

```diff
@@ -137,6 +137,19 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         }
     },
 
+    'Bryntum': {
+        id: 'bryntum',
+        icon: 'bryntum',
+        url: 'https://bryntum.com/',
+        npm: 'bryntum',
+        test: function(win) {
+            if(win.bryntum) {
```

Comment:
Please add an additional check here as [seen in other library tests](https://github.com/johnmichel/Library-Detector-for-Chrome/blob/6e7dcceb3471f10e34ac352bcf8b79f5908518ae/library/libraries.js#L592)